recipe/bld.bat

```diff
@@ -0,0 +1,22 @@
+mkdir build
+cd build
+
+cmake -G "%CMAKE_GENERATOR%" ^
```

Comment:
Use `NMake Makefiles` as the CMAKE Generator, this should fix the Azure issues

Reply:
@xhochy, thanks.
done and ready for review